src/main/java/uk/gov/hmcts/divorce/common/event/Applicant2RequestChanges.java

```diff
@@ -47,10 +45,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .name("Applicant 2 Request Changes")
             .description("Applicant 2 Requests changes to be made by Applicant 1")
             .grant(CREATE_READ_UPDATE, APPLICANT_2, SYSTEMUPDATE)
-            .grant(READ,
-                APPLICANT_1_SOLICITOR,
-                CASE_WORKER,
-                SUPER_USER)
```

Comment:
my ticket https://tools.hmcts.net/jira/browse/NFDIV-1793 implies that these READ grants should stay

Reply:
okay, I will revert that